src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -79,26 +82,34 @@ def train(self, runner):
                 such as snapshotting and sampler control.
 
         Returns:
-            The average return in last epoch cycle.
+            float: The average return in last epoch cycle.
 
         """
         last_return = None
 
-        for epoch in runner.step_epochs():
+        for _ in runner.step_epochs():
             for cycle in range(self.n_epoch_cycles):
                 runner.step_path = runner.obtain_samples(runner.step_itr)
                 last_return = self.train_once(runner.step_itr,
                                               runner.step_path)
+                if cycle == 0 and self.evaluate:
+                    tabular.record('TotalEnvSteps', runner.total_env_steps)
                 runner.step_itr += 1
 
         return last_return
 
     def log_diagnostics(self, paths):
-        """Log diagnostic information on current paths."""
+        """Log diagnostic information on current paths.
+
+        Args:
+            paths (list[dict]): A list of collected paths.
+
+        """
         self.policy.log_diagnostics(paths)
         self.qf.log_diagnostics(paths)
 
     def process_samples(self, itr, paths):
+        # pylint: disable=unused-argument
```

Comment:
a better pattern i have seen for this is to do `del unused_argument` which helps the static analyzer see that the arg is now out of scope and avoids the error. all in one line of code.